src/Application/Query/Collection.php

```diff
@@ -29,6 +29,11 @@ class Collection
     public $data;
 
     /**
+     * @param int   $page
+     * @param int   $limit
+     * @param int   $total
+     * @param array $data
```

Comment:
Unedded

Reply:
If you're using PHP-Cs-fixer, I think the "no_superfluous_phpdoc_tags" is needed. All of this params don't need to be documented.